app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
```

Comment:
We intended this to be a feature eventually to allow researchers to add people who can administer by not be shown as authors, I think since it fits better with our existing models we should implement it here. A UI challenge would be to clarify if clarification is necessary about why authors added would not be bibliographic.